src/cntk-train/src/main/scala/CNTKLearner.scala

```diff
@@ -16,12 +16,10 @@ import org.apache.spark.sql._
 import org.apache.spark.sql.types._
 
 trait CNTKParams extends MMLParams {
-
-  // This is only needed until Train* accepts CNTKLearner instead of CL acting like Train*
-  val labelsColumnName = StringParam(this, "labelsColumnName", "Label col", "labels")
+  val labelsColumnName = StringParam(this, "labelsColumnName", "label column name", "labels")
   def getLabelsColumnName: String = $(labelsColumnName)
 
-  val featuresColumnName = StringParam(this, "featuresColumnName", "feats col", "features")
+  val featuresColumnName = StringParam(this, "featuresColumnName", "features column name", "features")
```

Comment:
Sentence case for docs please :-)

Reply:
done, Eli has already modified the other params